src/components/MessageHTMLBody.vue

```diff
@@ -110,25 +110,18 @@ export default {
 			iframeDoc
 				.querySelectorAll('[data-original-style]')
 				.forEach((node) => node.setAttribute('style', node.getAttribute('data-original-style')))
-
+		},
+		async onShowBlockedContent() {
+			this.displayIframe()
 			this.hasBlockedContent = false
```

Comment:
this one can also move into the `displayIframe` method as it's used here and below :)